docs/REST API/rest-api.md

```diff
@@ -5752,6 +5753,72 @@ If a query has no results stored, then `results` will be an empty array:
 
 > Note: osquery scheduled queries do not return errors, so only non-error results are included in the report. If you suspect a query may be running into errors, you can use the [live query](#run-live-query) endpoint to get diagnostics.
 
+### Get query report for one host
+
+Returns a query report for a single host.
+
+`GET /api/v1/fleet/hosts/{id}/queries/{query_id}`
+
+#### Parameters
+
+| Name      | Type    | In    | Description                                |
+| --------- | ------- | ----- | ------------------------------------------ |
+| id        | integer | path  | **Required**. The ID of the desired host.          |
+| query_id  | integer | path  | **Required**. The ID of the desired query.         |
+
+#### Example
+
+`GET /api/v1/fleet/hosts/123/queries/31`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "query_id": 31,
+  "host_id": 1,
+  "host_name": "foo",
+  "last_fetched": "2021-01-19T17:08:31Z",
+  "report_clipped": false,
+  "results": [
+    {
+      "columns": {
+        "model": "USB 2.0 Hub",
+        "vendor": "VIA Labs, Inc."
+      }
+    },
+    {
+      "columns": {
+        "model": "USB Keyboard",
+        "vendor": "VIA Labs, Inc."
+      }
+    },
+    {
+      "columns": {
+        "model": "USB Reciever",
+        "vendor": "Logitech"
+      }
+    }
+  ]
+}
+```
+
+If a query has no results stored for the specified host, then `results` will be an empty array:
+
+```json
+{
+ "query_id": 31,
+  "host_id": 1,
+  "host_name": "foo",
+  "last_fetched": "2021-01-19T17:08:31Z",
+  "report_clipped": false,
+  "results": []
+}
```

Comment:
please add `discard_data`, `interval`, `query` to this response @mostlikelee

Reply:
@jacobshandling @mostlikelee just occurred to me after looking at this again: from a pure API user perspective, might make sense to have the UI do a separate call to [Get query](https://fleetdm.com/docs/rest-api/rest-api#get-query) to get that information, to keep this endpoint focused on the report.

Reply:
@rachaelshaw sounds good

Reply:
agree.  in that case, no changes needed to the backend PR